jupyter_server_config.json

```diff
@@ -7,32 +7,40 @@
       "notebook-starter": {
         "type": "notebook",
         "src": "./examples/Starter Notebook.ipynb",
-        "label": "Starter Notebook",
-        "description": "A notebook that is also a starter"
+        "label": "ABC Starter Notebook",
```

Comment:
what is ABC?

Reply:
actually, in general, please roll back all of the changes to these config files, unless it's to add _one_ example which uses an existing category, as their data-driven position is used in the acceptance tests.

Reply:
Done, see: https://github.com/deathbeds/jupyterlab-starters/pull/78